doc/pages/commands.asciidoc

```diff
@@ -11,24 +11,27 @@ command *q!* has to be used). Aliases are mentionned below each commands.
 
 == Files and Buffers
 
+For the following *write* commands, the *-sync* switch forces the synchronization
+of the file onto the filesystem
+
 *change-directory* [<directory>]::
     *alias* cd +
-    change the current directory to *directory*, or the home directory if
```

Comment:
Actually the stars should say, they're here on purpose, the rest of the file incorrectly uses brackets around parameter names.

It's the same convention as with man pages, parameters are highlighted in the description.